cmd/lotus-miner/sectors.go

```diff
@@ -1119,31 +1119,21 @@ var sectorsExtendCmd = &cli.Command{
 	ArgsUsage: "<sectorNumbers...>",
 	Flags: []cli.Flag{
 		&cli.Int64Flag{
-			Name:     "new-expiration",
-			Usage:    "new expiration epoch",
-			Required: false,
+			Name:  "new-expiration",
+			Usage: "new expiration epoch",
 		},
 		&cli.BoolFlag{
-			Name:     "v1-sectors",
-			Usage:    "renews all v1 sectors up to the maximum possible lifetime",
-			Required: false,
-		},
-		&cli.Int64Flag{
```

Comment:
I don't think we should have dropped this?

Reply:
I don't think this is coming from this PR?

Reply:
It's also fixed in the branch apparently.

Reply:
Probably because the branch was rebased.